src/services/token.service.js

```diff
@@ -0,0 +1,52 @@
+import { Token } from '../models/token.js';
+
+const save = async (userId, refreshToken) => {
+  const token = await Token.findOne({ where: userId });
+
+  if (token) {
+    token.refreshToken = refreshToken;
+    await token.save();
+
+    return;
+  }
+
+  await Token.create({ userId, refreshToken });
+};
+
+const saveResetToken = async (userId, resetToken) => {
+  const token = await Token.findOne({ where: userId });
```

Comment:
Similar to line 4, the `where` clause should be an object with the `userId` field. It should be `{ where: { userId } }` instead of `{ where: userId }`.